pkg/controller/workspace/utils/utilities.go

```diff
@@ -0,0 +1,33 @@
+package workspace
+
+import (
+	"strings"
+	"k8s.io/client-go/discovery"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func Join(sep string, parts ...string) string {
+	return strings.Join(parts, sep)
+}
```

Comment:
Why do we need this function?

Reply:
I think we don't. See https://github.com/che-incubator/che-workspace-crd-operator/pull/3/files#r363515566

Reply:
Already removed